src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleDifferentStyles.java

```diff
@@ -10,50 +10,53 @@
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
 
 @Deprecated
-@SomeArrays(pooches={DOGS.LEO}) // violation
-@SuppressWarnings({""}) // violation
+@SomeArrays(pooches={DOGS.LEO}) // violation 'Annotation style must be 'COMPACT_NO_ARRAY''
+@SuppressWarnings({""}) // violation 'Annotation style must be 'COMPACT_NO_ARRAY''
 public class InputAnnotationUseStyleDifferentStyles
 {
 
 }
 
-@SomeArrays(pooches={DOGS.LEO}, um={}, duh={"bleh"}) // violation
+@SomeArrays(pooches={DOGS.LEO}, // violation 'Annotation style must be 'COMPACT_NO_ARRAY''
+        um={}, duh={"bleh"})
```

Comment:
use `// violation below`

Reply:
Done. Code formatting has been rewritten to original form. Violation message has been rewritten accordingly.